core/tests/protractor/cacheSlugs.js

```diff
@@ -56,7 +56,9 @@ describe('Cache Slugs', function() {
   it('should check that errors get logged for missing resources', function() {
     browser.get(ERROR_PAGE_URL_SUFFIX);
     var expectedErrors = [
-      'http://localhost:9001/build/fail/logo/288x128_logo_white.png'
+      'http://localhost:9001/build/fail/logo/288x128_logo_white.png',
+      // Warning fired by synchronous AJAX request for constatns
```

Comment:
"constants" is misspelled.

Also, can you extend this comment to give a bit more context/detail, and/or a link to a reference, so that someone who doesn't have background understands what's going on here? Thanks!